plugins/repository-azure/src/main/java/org/elasticsearch/cloud/azure/storage/AzureStorageService.java

```diff
@@ -37,9 +37,12 @@
     final class Storage {
         public static final String PREFIX = "cloud.azure.storage.";
         @Deprecated
-        public static final String ACCOUNT = "cloud.azure.storage.account";
+        public static final String ACCOUNT_DEPRECATED = "cloud.azure.storage.account";
         @Deprecated
-        public static final String KEY = "cloud.azure.storage.key";
+        public static final String KEY_DEPRECATED = "cloud.azure.storage.key";
```

Comment:
Actually the filter is applied with a wildcard:

```java
        settingsFilter.addFilter("cloud.azure.storage.*");
```

I think that's why I did not explicitly filter it.  Does it make sense?